backend/typescript/models/scheduling.model.ts

```diff
@@ -0,0 +1,38 @@
+import { Column, DataType, Model, Table } from "sequelize-typescript";
+import { Status } from "../types";
+
+@Table({ tableName: "scheduling" })
+export default class Scheduling extends Model {
+  @Column({ type: DataType.INTEGER })
+  id!: number;
+
+  @Column({ type: DataType.INTEGER })
+  donor_id!: string;
```

Comment:
```suggestion

  @Column({ type: DataType.INTEGER })
  @Unique 
  donor_id!: string;
```

Reply:
I don't think the donor_id would be unique, would it? Each donor can schedule multiple drop offs right

Reply:
Yes donor_id should not be unique